packages/actor-extract-links-extract-tree/lib/SolutionRange.ts

```diff
@@ -0,0 +1,150 @@
+const nextUp = require('ulp').nextUp;
```

Comment:
This feels like a hacky approach.
Instead of this, can we make use of inclusive/exclusive flags for the solution range max/min?

Reply:
Sure, it fits, but it's a very nasty solution 😛 
Just using boolean to indicate inclusive/exclusive borders would be much cleaner, and easier for debugging.